src/cascadia/TerminalSettingsEditor/GlobalAppearance.cpp

```diff
@@ -195,4 +200,61 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         }
     }
 
+    // Function Description:
+    // - Updates the list of all themes available to choose from.
+    void GlobalAppearance::_UpdateThemeList()
+    {
+        // Surprisingly, though this is called every time we navigate to the page,
+        // the list does not keep growing on each navigation.

```

Comment:
This comment has a VERY threatening aura! I am almost certain it will break at some point...